### ISSUE TITLE
What about amount of data?

### ISSUE THREAD

**Issue by Phil353556** [yanniedog/flightaware-contribution#2]
Hello,
Thanks for sharing this repo, 
I tried it, works
However there is no a computed value about "_how much of your ADSB receiver data FlightAware are actually using._" 

It is a display of flights.
screenshots: data from 09/07/2024      Tue 01:39:28 AM to 09/07/2024      Tue 02:18:49 AM
but it is 08:46AM and there are lots of  flights since 02:18:49 AM 
what am i doing wrong?
Thanks

![image](https://github.com/yanniedog/fa/assets/64729485/f4c9e74a-7f7d-4bd9-b3e2-78cd58d9f14d)
![image](https://github.com/yanniedog/fa/assets/64729485/be8ef78f-bc66-4714-a26b-fa628d9b5fa2)

**Comment by yanniedog** [comment]
Your ADSB receiver logs a lot of flight positions.
You can retrieve this log directly from your receiver, if you wish.

All of the flight positions your receiver detects are uploaded to Flight Aware.

However, Flight Aware only uses a small proportion of the flight positions you upload. Most of the data you provide is discarded.

This repo gives users a way to record every instance where Flight Aware actually uses flight position data you provide it.

In other words.

This script will show you any time one of your detected "flight positions" is inserted into an aircraft's official flight path.

**Comment by yanniedog** [comment]
With regards to the time mismatch: please check whether the times you're seeing are in a different timezone
